src/rust/ensogl/lib/select/src/entry.rs

```diff
@@ -0,0 +1,310 @@
+//! A single entry in Select
+use crate::prelude::*;
+
+use ensogl_core::application::Application;
+use ensogl_core::data::color;
+use ensogl_core::display;
+use ensogl_text as text;
+use ensogl_text::buffer::data::unit::Bytes;
+use ensogl_text::buffer::data::range::Range as TextRange;
```

Comment:
Remove these imports. `ensogl_text` reexports a lot of things, so you can use `text::Bytes` or `text::Range`. If you want to import bytes, use shorter imort instead.